flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartitionTest.java

```diff
@@ -252,36 +306,24 @@ public void testReleaseWhileWriting() throws Exception {
 
     @Test
     public void testReleaseWhileReading() throws Exception {
```

Comment:
I think this can be a UT of `SortMergeResultPartitionReadScheduler` and then we can remove `SortMergeResultPartition#getReadScheduler()`.